low-level/sspl-ll.spec

```diff
@@ -140,12 +141,9 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 [ -f $SSPL_DIR/sspl_init ] ||
     ln -s $SSPL_DIR/bin/sspl_provisioner_init $SSPL_DIR/sspl_init
 
-# Creating softlink under site-packages to use sspl module easier way.
-PYTHON_BASE_DIR=/usr/lib/python3.6/site-packages/cortx/sspl
+# Creating softlink under sspl/bin s to use sspl module easier way.
 [ -d "${SSPL_DIR}" ] && {
-    mkdir -p $PYTHON_BASE_DIR
-    ln -sf $SSPL_DIR/bin $PYTHON_BASE_DIR/bin
-    ln -sf $SSPL_DIR/low-level $PYTHON_BASE_DIR/lowlevel
+    ln -sf $SSPL_DIR/low-level/* $SSPL_DIR/bin/
```

Comment:
Linking everything under low-level to bin? Only CLIs, binaries should be in bin dir

Reply:
Done